pkg/internal/apply/testdata/crds/addonslayers_crd.yaml

```diff
@@ -0,0 +1,155 @@
+
```

Comment:
Kubebuilder builds the crd yaml in config/crd/bases, maybe use that instead of duplicating?